core-web/libs/sdk/uve/src/lib/types/editor/public.ts

```diff
@@ -138,3 +140,46 @@ export enum DotCMSUVEAction {
      */
     NOOP = 'noop'
 }
+
+/**
+ * The contentlet has the main fields and the custom fields of the content type.
+ *
+ * @template T - The custom fields of the content type.
+ */
+export type Contentlet<T> = T & ContentTypeMainFields;
+
+/**
+ * Available events in the Universal Visual Editor
+ * @enum {string}
+ */
+export enum UVEEventType {
+    /**
+     * Triggered when page data changes from the editor
+     */
+    CONTENT_CHANGES = 'changes',
+
+    /**
+     * Triggered when the page needs to be reloaded
+     */
+    PAGE_RELOAD = 'page-reload',
```

Comment:
Isn't this part of the deprecation?

Reply:
If so... we can marke it as deprecated here too.

Reply:
We discussed this and we have to make some changes in the UVE, before deleting this one, it will come when we connect the UVE using GraphQL